react_main/src/pages/Game/Game.jsx

```diff
@@ -1816,6 +1847,17 @@ export function PlayerRows(props) {
       }
     }
 
+    let avatarId;
+
+  if (player !== undefined) {
```

Comment:
when would player be undefined?

avatarId = player.anonId ?? player.userId

Reply:
Not sure? There's quite a few things left in, probably don't need this.